spec/support/wait_for_ajax.rb

```diff
@@ -0,0 +1,19 @@
+module WaitForAjax
+  def wait_for_ajax(count = 1)
+    page.execute_script 'window._ajaxCalls = 0, window._ajaxCompleteCounter = function() { window._ajaxCalls += 1; }'
```

Comment:
Line is too long. [117/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.